client/containers/Pub/PubSubmissionHeader/spubHeader.scss

```diff
@@ -0,0 +1,72 @@
+@import 'styles/variables.scss';
+@import './themes.scss';
+
+$header-margin-top: 0px;
+$header-margin-bottom: 26px;
+$header-margin-bottom-mobile: 13px;
+$header-margin-total: $header-margin-top + $header-margin-bottom;
+$header-margin-total-mobile: $header-margin-top + $header-margin-bottom-mobile;
+$title-elements-spacing: 15px;
+$mobile-bottom-buttons-padding-top: 15px;
+$mobile-bottom-buttons-spacing: 10px;
+
+.pub-header-component {
+	position: relative;
+	z-index: 20;
+	border-bottom: 1px solid var(--community-accent-dark);
+	padding: 200px;
+
+	h1,
+	h2,
+	h3,
+	h4,
+	h5,
+	h6 {
+		color: inherit;
+	}
+
+	h1 {
+		margin: 0;
+
+		font-family: $header-font;
+		font-size: 40px;
+		font-style: normal;
+		font-weight: 400;
+		line-height: 56px;
+		letter-spacing: 0em;
+		text-align: left;
+
+		box-decoration-break: clone;
+		@include smaller-viewport {
+			font-size: 32px;
+		}
+	}
+}
+
+.bp3-tabs {
```

Comment:
All of these `.bp3-xyz` classes need to be scoped inside the (to be renamed) `pub-header-component` class up there, otherwise they'll target every control of this type in the application.